build-packages.sh

```diff
@@ -122,6 +122,7 @@ echo "Versioning Diff:"
 git diff | cat
 
 # Commit changes
+git add .
```

Comment:
```suggestion
git add maven-settings.xml
```

I would make this more specific so we know we're only adding this file, and not any other "junk"

Reply:
Are we okay with hard-coding the name of the settings file like this? It seems unlikely to change so it might be OK. We'd have to update it in a ton of other places if we ever changed it.

Other options would be to pass in the name of the file as a param (seems like overkill) or to try to copy it out of the .m2, and control the name as a result (is this feasible)?

Reply:
> Are we okay with hard-coding the name of the settings file like this? It seems unlikely to change so it might be OK. We'd have to update it in a ton of other places if we ever changed it.
> 
> Other options would be to pass in the name of the file as a param (seems like overkill) or to try to copy it out of the .m2, and control the name as a result (is this feasible)?

I think you said it already, its unlikely to change so its probably OK - Copying it out of ~/.m2/settings.xml is also viable.